.travis.yml

```diff
@@ -1,9 +1,15 @@
 language: rust
 
-services:
-    - "mongodb"
+before_script:
+    - wget https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-3.0.3.tgz
+    - tar xvf mongodb-linux-x86_64-3.0.3.tgz
+    - mv mongodb-linux-x86_64-3.0.3 3.0.3
+    - mkdir -p /data/db
+
+env:
+  - RUST_TEST_THREADS=1
```

Comment:
Oops, sorry, I'm used to two-spaces for YAML, but the first travis commit used four, so I keep forgetting to change mine.